gleam/Dockerfile

```diff
@@ -0,0 +1,18 @@
+{{#language.image}}
+  FROM gleamlang/gleam
```

Comment:
is there a maintained version of a docker image ?

Reply:
According to https://github.com/gleam-lang/gleam/pkgs/container/gleam it should be 
```suggestion
  FROM ghcr.io/gleam-lang/gleam:v1.6.3-elixir
```

Reply:
or maybe 
```dockerfile
FROM ghcr.io/gleam-lang/gleam:v1.6.3-erlang
```

Reply:
I wish somebody using gleam would give us hints